### PR TITLE
fix: update changelog list when preparing release

### PR DESCRIPTION
**Pull request by RafaelGSS**
Refs: https://github.com/nodejs/node-core-utils/issues/686

### CODE REVIEW

File: lib/prepare_release.js [hunk 1]
```diff
@@ -386,9 +386,10 @@ export default class ReleasePreparation {
                           ` ${releaseInfo}, @${username}\n`;
 
     const topHeader =
-      `<a href="#${lastRef.substring(1)}">${lastRef.substring(1)}</a><br/>`;
+      `<b><a href="#${lastRef.substring(1)}">${lastRef.substring(1)}</a></b><br/>`;
```
Comment (richardlau) [inline lib/prepare_release.js:389]: We may need to make this optional for it to work with all release lines.
Compare
- https://github.com/nodejs/node/blob/main/doc/changelogs/CHANGELOG_V16.md?plain=1
- https://github.com/nodejs/node/blob/main/doc/changelogs/CHANGELOG_V18.md?plain=1
- https://github.com/nodejs/node/blob/main/doc/changelogs/CHANGELOG_V20.md?plain=1

Comment (RafaelGSS) [reply lib/prepare_release.js:389]: what if we just drop the <b> on v20?

Comment (richardlau) [reply lib/prepare_release.js:389]: > what if we just drop the `<b>` on v20?

SGTM as well.

File: lib/prepare_release.js [hunk 2]
```diff
@@ -386,7 +386,7 @@ export default class ReleasePreparation {
                           ` ${releaseInfo}, @${username}\n`;
 
     const topHeader =
-      `<a href="#${lastRef.substring(1)}">${lastRef.substring(1)}</a><br/>`;
+      `<a href="#${lastRef.substring(1)}">${lastRef.substring(1)}</a>`;
```
Comment (RafaelGSS) [inline lib/prepare_release.js:389]: now it should match with or without <b>

**Review by richardlau — APPROVED** [review]

**Pull request merged**